server/core/runtime/post_workflow_hook_runner.go

```diff
@@ -47,8 +54,25 @@ func (wh DefaultPostWorkflowHookRunner) Run(ctx models.WorkflowHookCommandContex
 	if err != nil {
 		err = fmt.Errorf("%s: running %q in %q: \n%s", err, command, path, out)
 		ctx.Log.Debug("error: %s", err)
-		return "", err
+		return "", "", err
+	}
+
+	// Read the value from the "outputFilePath" file
+	// to be returned as a custom description.
+	var customStatusOut []byte
+	if _, err := os.Stat(outputFilePath); err == nil {
+		var customStatusErr error
+		customStatusOut, customStatusErr = os.ReadFile(outputFilePath)
+		if customStatusErr != nil {
+			err = fmt.Errorf("%s: running %q in %q: \n%s", err, command, path, out)
+			ctx.Log.Debug("error: %s", err)
+			return "", "", err
+		}
 	}
+
+	wh.OutputHandler.SendWorkflowHook(ctx, string(out), false)
+	wh.OutputHandler.SendWorkflowHook(ctx, "", true)
```

Comment:
nitpick: can we format `string(out)` with a newline to avoid another function call?